lib/chat_notifier.rb

```diff
@@ -105,8 +130,7 @@ def chat_users
   end
 
   def rest_of_the_channel
-    chat_users
```

Comment:
this is changing nothing but formatting right?

Reply:
Yeah, hard to see in the github diff

Reply:
We should have ruby formatting by the end of the day on chat 🎉

Reply:
Yay!!!